docs/source/getting_started/installation.md

```diff
@@ -51,41 +51,41 @@ Prior to version 3.1, the `elyra` package included all dependencies. Subsequent
 If you use `pip`, install Elyra with:
 
 ```bash
-pip3 install --upgrade elyra[all] && jupyter lab build
+pip3 install --upgrade elyra[all]
```

Comment:
Just curious, do we still need to specifically add `[all]` to install all packages by default?

Reply:
@akchinSTC @lresende you two know more about pip install details than I do

Reply:
> Just curious, do we still need to specifically add [all] to install all packages by default?

The `[all]` that is referenced here is really about how the [`extras_require` entry](https://github.com/elyra-ai/elyra/blob/43e6d9869c8e5bf2dd8213e8042c2d120d196fe7/setup.py#L110-L113) works wrt to pip.  In this case `[all]` is a "meta-entry" that encompasses the [4 separately installable "sub-packages"](https://github.com/elyra-ai/elyra/blob/43e6d9869c8e5bf2dd8213e8042c2d120d196fe7/setup.py#L43-L54), and not related to the changes in this PR.